### DISCUSSION TITLE
script to calculate standard deviation maps?

### DISCUSSION THREAD

**Discussion by mbnebel** [thebrisklab/RestingStateMultiband#4] [Q&A] [answered]
Quick question: where do you calculate the *_stdev.nii.gz files that 2_CreateAveSDMaps.m averages?

**Comment by BenjaminRisk** [comment, marked as answer]
It's in the shell script calculated by AFNI, line 436 in preprocessing_singlesubj_3dTproject.sh 

$afnidir/3dTstat -stdev -prefix ${func_dir}/${rest}_pp_9p_sm_stdev.nii.gz ${func_dir}/${rest}_pp_9p_sm.nii.gz  -overwrite
